refinery/data_set_manager/utils.py

```diff
@@ -730,13 +730,23 @@ def get_owner_from_assay(uuid):
     except (Study.DoesNotExist,
             Study.MultipleObjectsReturned) as e:
         logger.error(e)
-        return "Error: Invalid uuid"
+        return None
```

Comment:
If `None` is really what we want to be returning upon error conditions, then there could be an assignment at the top: `owner = None`, and the final `return owner` would take the place of the other 3 `return None`.

Reply:
@scottx611x I thought returning None made more sense than returning an error message. It's only used in one place to check against the request.owner. Thoughts?
The refactoring makes sense.

Reply:
Sounds fine to me, the loggers are probably enough

Reply:
Can't do the refactoring. The trys depend on the previous info.

Reply:
@jkmarx What about a simplification like:

```suggestion
    owner = None
    try:
        study = Study.objects.get(assay__uuid=uuid)
    except (Study.DoesNotExist,
            Study.MultipleObjectsReturned) as e:
        logger.error(e)
    else:
        try:
            dataset = study.get_dataset()
        except RuntimeError as e:
            logger.error(e)
        else:
            owner = dataset.get_owner()
    return owner
```

Reply:
Not keen on a nested try.
How about:
```
    try:
        study = Study.objects.get(assay__uuid=uuid)
    except (Study.DoesNotExist,
            Study.MultipleObjectsReturned) as e:
        logger.error(e)
        return None

    try:
        data_set = study.get_dataset()
    except RuntimeError as e:
        logger.error(e)
        return None

    return data_set.get_owner()
```

Reply:
@scottx611x